index.js

```diff
@@ -1,53 +1,45 @@
 // Load dotenv first
 require('dotenv').config({ silent: true });
-
-const utility = require('./src/utility');
+const SubmissionRepos = require('./src/Whitelist');
+const BlackListed = require('./src/Blacklist');
```

Comment:
wait you changed the names here but the file names themselves are not changed?

Reply:
ahh my git was ignoring case changes

Reply:
ah unix system